src/main/scala/no/ndla/conceptapi/ConceptSwagger.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Part of NDLA draft_api.
+ * Copyright (C) 2019 NDLA
+ *
+ * See LICENSE
+ */
+/*
```

Comment:
Er denne noe du skal bruke senere? For nå er det bare en utkommentert fil.

Reply:
Vi skal jo ha swagger for concept-api også så du kan like gjerne ordne opp i denne.